service/features/service.feature

```diff
@@ -1572,12 +1572,10 @@ Feature: VxFlex OS CSI interface
 
   Scenario: Call NodeGetInfo with zone label
     Given a VxFlexOS service
-    And I use config "multi_az"
+    And I use config <config>
     When I call NodeGetInfo with zone labels
     Then a valid NodeGetInfo is returned with node topology
-
-  Scenario: Call NodeGetInfo with zone label with invalid config
```

Comment:
**question:** Do we no longer need this test?

Reply:
This test was added before when a zone label other than "zone.csi-vxflexos.dellemc.com" would be considered an invalid config. With custom labels, this test is no longer valid.